azure-iot-device/azure/iot/device/iothub/client_event.py

```diff
@@ -0,0 +1,15 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+NEW_SASTOKEN_REQUIRED = "NEW_SASTOKEN_REQUIRED"
+CONNECTION_STATE_CHANGE = "CONNECTION_STATE_CHANGE"
+BACKGROUND_EXCEPTION = "BACKGROUND_EXCEPTION"
+
+
+class ClientEvent(object):
+    def __init__(self, name, values_for_user=[]):
```

Comment:
>values_for_user [](http://example.com/codeflow?start=29&length=15)

not sure what this is.  It needs a better name.  If these are args that get passed to the user, maybe call it args or args_for_event_callback or something like that.